iroha-lib/model/generators/CommandGenerator.cpp

```diff
@@ -0,0 +1,258 @@
+#include "CommandGenerator.hpp"
+#include "model/converters/json_transaction_factory.hpp"
+
+
+namespace iroha_lib
+{
+
+std::shared_ptr<iroha::protocol::Command> CommandGenerator::generateAddAssetQuantity(const std::string& asset_id,
+                                                                                     const std::string& amount)
+{
+    iroha::protocol::AddAssetQuantity addAssetQuantity;
+    addAssetQuantity.set_asset_id(asset_id);
+    addAssetQuantity.set_amount(amount);
+
+    auto cmd = iroha::protocol::Command();
+    cmd.set_allocated_add_asset_quantity(new iroha::protocol::AddAssetQuantity(addAssetQuantity));
+    return generateCommand<iroha::protocol::Command>(cmd);
+}
+
+std::shared_ptr<iroha::protocol::Command> CommandGenerator::generateAddPeer(const std::string& address,
+                                                                            const std::string& pubkey,
+                                                                            const std::optional<std::string>& tls_certificate,
+                                                                            bool syncing_peer)
+{
+    iroha::protocol::AddPeer pb_add_peer;
+    auto peer = pb_add_peer.mutable_peer();
+
+    iroha::protocol::Peer res;
+    res.set_address(address);
+    res.set_peer_key(iroha::hexstringToArray<iroha::pubkey_t::size()>(pubkey).value().to_hexstring());
+
+    if (tls_certificate.has_value())
```

Comment:
Please use a consistent style for braces. K&R don't condone this form of omission.

Reply:
Done.